src/main/java/io/openliberty/tools/intellij/LibertyExplorer.java

```diff
@@ -48,23 +51,37 @@ public class LibertyExplorer extends SimpleToolWindowPanel {
 
     public LibertyExplorer(@NotNull Project project) {
```

Comment:
Constructors normally complete their initialization synchronously, allowing the creator of that object to reliably read the initial state of the object. With this change which dispatching the initialization on to other threads, the constructor can (and likely will return) before either the content or toolbar is set. With the inner calls to `invokeLater()` it seems the content and toolbar could be set in any order. Did you verify that no users of this object assume that these fields have been set within the constructor? A demo which shows this working would help.

Reply:
I will conduct the demo today to demonstrate the functionality, after which we can finalize

Reply:
Thanks for the demo today. It certainly appears things are working as expected.

Reply:
@vaisakhkannan Thanks for verifying.